mainapp/migrations/0053_add_missing_indices_v1.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.1 on 2018-08-19 04:28
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mainapp', '0052_auto_20180819_0128'),
+    ]
+
+    operations = [
+        migrations.RunSQL("CREATE INDEX request_combined_index ON mainapp_request (district, requestee, requestee_phone, location)"),
```

Comment:
CREATE INDEX on postgres obtains lock on the entire table. It's better to use CREATE INDEX CONCURRENTLY.
ref: https://www.postgresql.org/docs/10/static/sql-createindex.html